src/util.js

```diff
@@ -430,7 +430,8 @@ const util = {
       return os.cpus().length;
     }
 
-    return navigator.hardwareConcurrency || 1;
+    // eslint-disable-next-line no-mixed-operators
+    return typeof navigator !== 'undefined' && navigator.hardwareConcurrency || 1;
```

Comment:
```suggestion
    return (typeof navigator !== 'undefined' && navigator.hardwareConcurrency) || 1;
```

Reply:
The latter isn't quite the same in case `navigator` exists but `navigator.hardwareConcurrency` doesn't, but the former works, thanks :+1: